### ISSUE TITLE
Generate example app inside our repo on every change

### ISSUE THREAD

**Issue by goldbergyoni** [practicajs/practica#191]
**Need -** It could be great to show an example app immediately in our repo without the need to use the CLI. For this, in every CI run, we can invoke our CLI, generate app to a local folder [root]/example-app/express-pg, then commit back to the repo

**Comment by rluvaton** [comment]
I think that instead of commuting back to the repo, we will push to different repositories that also can be used as GitHub templates

**Comment by goldbergyoni** [comment]
For longer term, you're right

For short term, we need the stars... If /practica/express-example become popular - we need all the stars in one repo

**Comment by trasherdk** [comment]
I would :star: both `example-app/express-pg` and `example-app/express-mysql` as separate repositories, even though I don't do `PG` :smiley:

**Comment by goldbergyoni** [comment]
@trasherdk Makes sense, we can reuse the ORMs capability of supporting two popular DBs...

Let's do it!
